### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by pyr0gan**
- Fixed broken links as per Google Doc shared by @yash-sharma1 
- Fixed some remaining HTML tags
- Fixed some typos and formatting

### CODE REVIEW

**Review by zurgl — CHANGES_REQUESTED** [review]
@pyr0gan Almost perfect, but I found one link which is not working please review.

File: network-documentation/avalanche/tutorials/create-a-dex-with-avalanche.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ A decentralized exchange is a network, where anyone can exchange cryptocurrency
 
 # Prerequisites
 
-You must have gone through this tutorial [Create a Local Test Network on Avalanche](https://learn.figment.io/network-documentation/avalanche/tutorials/platform-overview/create-a-local-test-network) and have performed a cross-chain swap via the [Transfer AVAX Between X-Chain and C-Chain](https://learn.figment.io/network-documentation/avalanche/tutorials/token-transfer-tutorials/transfer-avax-between-the-x-chain-and-c-chain) tutorial to get AVAX test tokens to your C-Chain address.
+You must have gone through this tutorial [Create a Local Test Network on Avalanche](https://learn.figment.io/tutorials/create-a-local-test-network) and have performed a cross-chain swap via the [Transfer AVAX Between X-Chain and C-Chain](https://learn.figment.io/network-documentation/avalanche/tutorials/token-transfer-tutorials/transfer-avax-between-the-x-chain-and-c-chain) tutorial to get AVAX test tokens to your C-Chain address.
```
[the remark this answers is not in the document]
Comment (pyr0gan) [reply network-documentation/avalanche/tutorials/create-a-dex-with-avalanche.md:7]: Fixed!

[the remark this answers is not in the document]
Comment (yash-sharma1) [reply network-documentation/avalanche/tutorials/create-a-dex-with-avalanche.md:7]: @pyr0gan At the second line, this [Transfer AVAX Between X-Chain and C-Chain] should be linked to this https://learn.figment.io/tutorials/transfer-avax-between-the-x-chain-and-c-chain

File: network-documentation/secret/tutorials/deploy-your-own-secret-nft.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+For a high level introduction to Non-Fungible-Tokens see the [first installment of this series](https://learn.figment.io/tutorials/create-your-first-secret-nft). In this tutorial we will download and compile the [snip721 reference implementation](https://github.com/baedrik/snip721-reference-impl), deploy it onto the secret testnet and interact with the contract, minting your own secret NFTs. Unlike the [previous tutorial](https://learn.figment.io/network-documentation/secret/tutorials/create-your-first-secret-nft) we will configure the contract ourselves and learn about access right management of secret contracts and tokens on the way.
```
Comment (yash-sharma1) [inline network-documentation/secret/tutorials/deploy-your-own-secret-nft.md:3]: Link to [previous tutorial] also need to be updated to https://learn.figment.io/tutorials/create-your-first-secret-nft

**Comment by yash-sharma1** [comment]
@pyr0gan left some comments for links to fix other than that all good & great job!

**Pull request merged**